src/net_processing.h

```diff
@@ -25,11 +25,14 @@ static const bool DEFAULT_PEERBLOOMFILTERS = false;
 static const bool DEFAULT_PEERBLOCKFILTERS = false;
 /** Threshold for marking a node to be discouraged, e.g. disconnected and added to the discouragement filter. */
 static const int DISCOURAGEMENT_THRESHOLD{100};
+/** -pingtimeout default */
+static const int64_t DEFAULT_PING_TIMEOUT = true;
```

Comment:
another comment on code that you simply moved-

I found this definition odd. Why is it defined as an int but set as a boolean? Also, why is it an `int64_t`, that seems like more space needed than for a simple bool value? 

also note that this could be constexpr, but I don't know if that actually impacts anything. I tried it out in [godbolt explorer](https://godbolt.org/z/61anP3) and looks like for such simple  logic, it compiled down to the same assembly regardless 🤷🏽‍♀️

Reply:
No, I added this code. It's an int because originally the config option was to set how long the timeout. I later changed the config option to be binary on/off (https://github.com/bitcoin/bitcoin/pull/20721#issuecomment-773227475), but failed to change the type to bool. Will fix.